src/arrays.js

```diff
@@ -29,6 +37,13 @@ const reduce = (elements, cb, memo = elements.shift()) => {
   // Combine all elements into a single value going from left to right.
   // Elements will be passed one by one into `cb`.
   // `memo` is the starting value.  If `memo` is undefined then make `elements[0]` the initial value.
+  let reduced = (memo === undefined) ? elements.shift() : memo;
+ 
+   for (let i = 0; i < elements.length; i++) {
+     reduced += elements[i];
```

Comment:
Instead of using `+=` we should be reassigning the `memo` variable.  `memo = cb(memo, elements[i]);`  `+=` gets the tests to pass (I'll be adding more in the future so this kind of a solution wouldn't work) but it is only adding numbers together or concatenating strings.  Reduce is intended to be more general than that